src/System.IdentityModel.Tokens.Jwt/JwtPayload.cs

```diff
@@ -534,11 +536,19 @@ internal static object GetClaimValueUsingValueType(Claim claim)
                 return JArray.Parse(claim.Value);
             }
 
+            if (claim.ValueType == JsonClaimValueTypes.JsonNull)
```

Comment:
This whole block is looking rife for a switch statement, but we don't need to change it right now.